P/PThreadPool/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "PThreadPool"
+version = v"0.0.20191029"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Maratyszcza/pthreadpool.git", "d465747660ecf9ebbaddf8c3db37e4a13d0c9103"),
+    GitSource("https://github.com/Maratyszcza/FXdiv.git", "b742d1143724d646cd0f914646f1240eacf5bd73"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd pthreadpool
+mkdir build
+cd build
+cmake \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DFXDIV_SOURCE_DIR=$WORKSPACE/srcdir/FXdiv \
+    -DPTHREADPOOL_LIBRARY_TYPE=shared \
+    -DPTHREADPOOL_BUILD_TESTS=OFF \
+    -DPTHREADPOOL_BUILD_BENCHMARKS=OFF \
+    ..
+cmake --build . -- -j $nproc
+make install
+install_license ../LICENSE
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+filter!(p -> !Sys.iswindows(p), platforms) # Windows fails to build
```

Comment:
https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=17902&view=logs&j=bf08fc6b-a6d6-514d-d910-d67f80a9f009&t=d1c794c2-60c3-5b4e-76f3-47455db146a8&l=232

Reply:
It's the same reason as for filtering out a few platforms for CPUInfo: I'm trying to match the exact version used by PyTorch v1.5.0, cf. https://github.com/pytorch/pytorch/tree/v1.5.0/third_party - even if a few (major) platforms don't build (out of the box).

Reply:
It was this commit: https://github.com/Maratyszcza/pthreadpool/commit/ebd50d0cfa3664d454ffdf246fcd228c3b370a11 (the commit following the 2019-10-29 version: https://github.com/Maratyszcza/pthreadpool/commit/d465747660ecf9ebbaddf8c3db37e4a13d0c9103)

Reply:
How about using https://github.com/Maratyszcza/pthreadpool/commit/ebd50d0cfa3664d454ffdf246fcd228c3b370a11 then?

Reply:
Will add that with #4566